packages/xod-arduino/src/Directives.rei

```diff
@@ -50,6 +50,17 @@ let doesRaiseErrors: code => bool;
  */
 let isDirtienessEnabled: (code, string) => bool;
 
+/**
+  Returns if `evaluateTmpl` is found in the C++ code
+ */
+let implementsEvaluateTmpl: code => bool;
+
+/**
+  Returns if a node declares that it needs values
+  of read-only inputs in the state constructor
+ */
+let wantsStateConstructorWithParams: code => bool;
```

Comment:
Can the doc comments describe the corresponding pragma in details. See comments for the functions above for the reference

Reply:
Done!